src/components/Common/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,114 @@
+/**
```

Comment:
The comments provided are not necessary for such a simple component. Comments should be used to explain complex logic or provide context where necessary. In this case, the code is straightforward and self-explanatory, so the comments can be omitted.

Reply:
I'll retain only essential comments for clarity.